cellrank/tl/estimators/_base_estimator.py

```diff
@@ -341,12 +341,9 @@ def compute_absorption_probabilities(
 
             Information on the :mod:`scipy` iterative solvers can be found in :func:`scipy.sparse.linalg` or for
             :mod:`petsc4py` solver `here <https://www.mcs.anl.gov/petsc/documentation/linearsolvertable.html>`__.
-
-            If is `None`, the solver is chosen automatically, depending on the problem size.
         use_petsc
             Whether to use solvers from :mod:`petsc4py` or :mod:`scipy`. Recommended for large problems.
-            If `None`, it is determined automatically. If no installation is found, defaults to
-            :func:`scipy.sparse.linalg.gmres`.
+            If no installation is found, defaults to :func:`scipy.sparse.linalg.gmres`.
```

Comment:
Did you check whether that makes it a lot slower on small problems? I know for sure that the direction method **will be* faster for small problems, the question is *how much*.

Reply:
```
# keys are subsets of the pancreas data, values are (mean,std) for 10 random subsets
PETSc + gmres
{100: (0.011416912078857422, 0.004280086204327923),
 250: (0.018981313705444335, 0.003186679585923787),
 500: (0.021230626106262206, 0.0012997817509389086),
 1000: (0.057153940200805664, 0.005951826256232277),
 1500: (0.09224464893341064, 0.011383153019094908),
 2000: (0.1188497543334961, 0.0013607115998565049),
 2500: (0.1569295644760132, 0.0022418019388140424)}

PETSc + direct
{100: (0.005582404136657715, 0.0013073040658704105),
 250: (0.007641458511352539, 0.0016195266670197747),
 500: (0.012239551544189453, 0.0015909780405909395),
 1000: (0.028612852096557617, 0.002168457370086139),
 1500: (0.07074854373931885, 0.012893574633071328),
 2000: (0.1430955410003662, 0.01825136893836883),
 2500: (0.2457975149154663, 0.024587187138362544)}

scipy + direct
{100: (0.0058349609375, 0.0017682622847889555),
 250: (0.014855360984802246, 0.01063481691736127),
 500: (0.01383976936340332, 0.0019364345745918484),
 1000: (0.033646798133850096, 0.009655263707504665),
 1500: (0.16855034828186036, 0.11022524525949293),
 2000: (0.1577007532119751, 0.08393314372050473),
 2500: (0.2498032808303833, 0.04451559792479094)}
```

Reply:
really well done @michalk8, thank you very much for looking into this and for running these comparisons!